README.md

```diff
@@ -169,12 +169,12 @@ $bob['firstName'] = 'Bob';
 $bob['email'] = 'bob@example.com';
 $datastore->insert($bob);
 
-// Fetch an entity
-$key = $datastore->key('Person', 'Bob');
+// Retrieve the entity we just created.
```

Comment:
Indeed. Do you think additional comments explaining why we're running the unnecessary lookup in this case (to demonstrate how it works) are useful?